CHANGES.rst

```diff
@@ -8,6 +8,10 @@
   - ``flycheck-go-build-install-deps`` turns on dependency installation for ``go test``
     as well as ``go build`` [GH-1003]
 
+- New syntax checkers:
```

Comment:
Ah, thanks, but could you just move this item above `- New features`?  Generally we list breaking changes, then new syntax checkers, and then other new features, because I think that support for new languages is more important to most users than other options.

@ibrahima Could you just make this final change?